en/_posts/2016-04-21-marathon-load-balancer.md

```diff
@@ -6,13 +6,18 @@ author: devatoria
 lang: en
 ---
 
-Marathon load balancer can be configured to use a specific vhost, so the URL to access your application will be more user-friendly. For example, let's say we have a nginx application and we want it to be available at `http://myapp.info`.
+The Marathon load balancer can be configured to use a specific VHost (virtual host), to make you application URL more user-friendly. For example, let's say we have a nginx application and we want it to be available at `https://myapp.info`.
 
-Considering your domain name is pointing to your load balancer URL (http://username.lb.sbg-1.containers.ovh.net), you just have to add the following labels:
+Considering your domain name is pointing to your load balancer URL (https://<username>.lb.sbg-1.containers.ovh.net), you just have to add the following labels:
 
 - **HAPROXY\_0\_MODE**: "http"
 - **HAPROXY\_0\_VHOST**: "myapp.info"
 
 ![Main interface](/kb/images/2016-04-21-marathon-load-balancer/haproxy.png)
 
-If you want more information about how to configure your HAProxy load balancer, please [read this documentation](https://github.com/mesosphere/marathon-lb#haproxy-configuration).
+If you want more information about how to configure your HAProxy load balancer, please read the [marathon-lb documentation](https://github.com/mesosphere/marathon-lb#haproxy-configuration). You'll see for example how to set weight on backends, how to enable sticky sessions on each backend.
+
+## Configuring ACLs 
+
+**Note**: there is an undocumented (for the moment) feature in marathon-lb: you can configure ACLs (Access Control List) on your applications, thus only allowing given IPs or networks to have access to your containers.
```

Comment:
Seems weird to speak of undocumented feature in... documentation ;) Do you mean upstream documentation ?